src/devices/Pn532/Pn532.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <LangVersion>7.3</LangVersion>
+    <RootNamespace>Iot.Device.Pn532</RootNamespace>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
+    </ProjectReference>
+    <PackageReference Include="System.IO.Ports" Version="4.6.0-preview6.19303.8" />
```

Comment:
this should be using the same version as Iot.Device.Bindings - adjust one or the other

Reply:
So "4.5.0" like in the Iot.Device.Bindings. But the 4.5.0 versio is not compatible with Linux serial port. So we should adjust the version in the IoT.Device.Bindigs to be compatible with Linux support. Or did I miss something?
So in short the 4.6.0-preview6.19303.8 or the 4.6.0-preview7.19362.9 or any of the 4.6.0-preview version.

Reply:
preview7 is latest for now